packages/client/src/devices/devices.ts

```diff
@@ -289,6 +300,8 @@ const watchForDisconnectedDevice = (
  * @angular It's recommended to use the [`DeviceManagerService`](./DeviceManagerService.md) for a higher level API, use this low-level method only if the `DeviceManagerService` doesn't suit your requirements.
  * @param deviceId$ an Observable that specifies which device to watch for
  * @returns
+ *
+ * @deprecated use the [new device API](https://getstream.io/video/docs/javascript/guides/camera-and-microphone/)
```

Comment:
Why not completely remove these APIs, as we are now presenting a better alternative?

Reply:
I think technically, that would be a breaking change, we export these methods, so if someone uses them, it will lead to an error on their side. That's why I only deprecated them

Reply:
With the previous device API, you needed to use these methods in your own code

Reply:
they were used in React's specific device management API before. Perhaps I should have removed them as part of #1127.

Reply:
I think we didn't know we won't need them. But in any case, I think it's fine to remove them in the next breaking release, in practice probably no one uses them.